doc/specs/vulkan/chapters/extensions.txt

```diff
@@ -344,21 +354,16 @@ endif::VK_VERSION_1_1,VK_KHR_get_physical_device_properties2[]
 == Extension Dependencies
 
 Some extensions are dependent on other extensions to function.
-To use extensions with dependencies, such _required extensions_ must: also
-be enabled through the same API mechanisms when creating an instance with
-flink:vkCreateInstance or a device with flink:vkCreateDevice.
+To enable extensions with dependencies, such _required extensions_ must:
+also be enabled through the same API mechanisms when creating an instance
+with flink:vkCreateInstance or a device with flink:vkCreateDevice.
 Each extension which has such dependencies documents them in the
 <<extensions, appendix summarizing that extension>>.
 
-[NOTE]
-.Note
-====
-The Specification does not currently include required extensions in Valid
-Usage statements for individual commands and structures, although it might
-do so in the future.
-Nonetheless, applications must: not use any extension functionality if
-dependencies of that extension are not enabled.
-====
```

Comment:
Not sure why you moved this note, I think its current location makes more sense.

Otherwise the proposed changes seem fine to me.

Reply:
I sorta repurposed it. It no longer describes extension dependencies (which are undefined behavior per VUs), so it no longer belongs in the Extension Dependencies chapter.

I made it instead to link to the Extension Implicit VU chapter.

Reply:
I've looked at your new language a lot, and perhaps I'm starting to see what your intention is, and how you're re-purposing the language.  It's using a non-defined, unique term ("necessary extensions"), that didn't convey to me what I now think you're trying to do.  I'll suggest some language to help convey what I think you want.

In addition, I want you to keep the first sentence of the old note here.  The 2nd sentence is repetitive with the paragraph above it; and is normative, which means it doesn't belong in a note.  Still, keep the first sentence (it serves a purpose internal to Khronos).

Reply:
The first sentence does not really make sense to me here.
It is already undefined behavior on `vkCreateInstance\Device`. So those commands are already undefined behavior, if you use null or invalide instace\device\child-of-those. It would make no sense to ever add Required Extension VUs to those, because we are already in undefined behavior teritory.